src/ui/Controllers/AuthController.cs

```diff
@@ -50,6 +51,13 @@ public ActionResult WhoAmI()
             await HttpContext.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
         }
 
+        [Authorize]
+        public ViewResult Unauthorized()
```

Comment:
Can we be a little bit more explicit with method name

ie 
There is an attribute that says you `authorise` but method name is `unauthorise`

Reply:
Good point! I'll do that...